src/Raven.Server/Documents/TombstoneCleaner.cs

```diff
@@ -132,7 +174,11 @@ internal TombstonesState GetState(bool addInfoForDebug = false)
             using (var tx = storageEnvironment.ReadTransaction())
             {
                 foreach (var tombstoneCollection in _documentDatabase.DocumentsStorage.GetTombstoneCollections(tx))
+                {
+                    if (tombstoneCollection != AttachmentsStorage.AttachmentsTombstones && tombstoneCollection != RevisionsStorage.RevisionsTombstones)
```

Comment:
Why aren't we checking those collections?

Reply:
done.